src/main/java/pl/coderslab/imageManager/image/ImageRepository.java

```diff
@@ -0,0 +1,9 @@
+package pl.coderslab.imageManager.image;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import pl.coderslab.imageManager.image.Image;
+
+public interface ImageRepository extends JpaRepository<Image, Long> {
+
+    public Image getByName(String name);
```

Comment:
hmm. ta metoda chyba nie jest potrzebna? Bo teraz nie zaciągałem kodu do intellij, ale spring data daje Ci już takie metody, tylko to się nazywa findByName. Czyli spróbuj wywalić tę metodę tutaj i robić findByName(jakaś_nazwa)

Reply:
Plus czemu 1 image a nie lista? Bo rozumiem że zakładasz że name jest unikalne, ale to trzeba oznaczyć w kodzie bo teraz może być taka sytuacja że wrzucić 2 image z tym samym imieniem i getByName się wywali bo będą 2 image a nie jeden. Tu masz jak to zrobić: 

https://stackoverflow.com/questions/15372654/uniqueconstraint-and-columnunique-true-in-hibernate-annotation

(@Column(unique = true)) na name